istio/README.md

```diff
@@ -3,17 +3,27 @@
 ## Overview
 
 Datadog monitors every aspect of your Istio environment, so you can:
-- Assess the health of Envoy and the Istio control plane with logs ([see below](#log-collection)).
-- Break down the performance of your service mesh with request, bandwidth, and resource consumption metrics ([see below](#metrics)).
+- Assess the health of Envoy and the Istio control plane with [logs](#log-collection).
+- Break down the performance of your service mesh with [request, bandwidth, and resource consumption metrics](#metrics).
 - Map network communication between containers, pods, and services over the mesh with [Network Performance Monitoring][1].
 - Drill into distributed traces for applications transacting over the mesh with [APM][2].
 
-To learn more about monitoring your Istio environment with Datadog, [see the Istio blog][3].
+To learn more about monitoring your Istio environment with Datadog, [see the Monitor blog post][3].
 
 ## Setup
 
 Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][4] for guidance on applying these instructions.
 
+### Prerequisites
+
+This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][25].
+
+When you enable the `use_openmetrics` configuration option, the Istio integration uses the OpenMetrics V2 implementation of the check. In OpenMetrics V2, metrics are submitted more accurately by default and behave closer to Prometheus metric types. For example, Prometheus metrics ending in  `_count` and `_sum` are submitted as `monotonic_count` by default.
```

Comment:
Should we replace "OpenMetrics V2" with "latest" in this paragraph? To reduce the proliferation of terms used to refer to it. Also, I think that we should be able to remove the second sentence of this paragraph because that's common to all integrations with the V2 / V1 divide, and if anything we should probably consider adding this to the [new guide on the topic](https://github.com/DataDog/documentation/blob/master/content/en/integrations/guide/versions-for-openmetrics-based-integrations.md).

Reply:
```suggestion
When you enable the `use_openmetrics` configuration option, the Istio integration uses the OpenMetrics implementation of the check. In OpenMetrics, metrics are submitted more accurately by default and behave closer to Prometheus metric types. For example, Prometheus metrics ending in  `_count` and `_sum` are submitted as `monotonic_count` by default.
```